lib/ethpy/ethpy/hyperdrive/state/__init__.py

```diff
@@ -1,6 +1,5 @@
 """Hyperdrive state classes and conversion helper functions."""
+from hypertypes.fixedpoint_types import FeesFP, PoolConfigFP, PoolInfoFP
```

Comment:
I don't think this should be here.

Reply:
everywhere that this is used (`FeesFP, PoolConfigFP, PoolInfoFP` are imported from `ethpy.hyperdrive.state` should be rewritten to directly import it from `hypertypes.fixedpoint_types`

Reply:
good catch, this happened when vscode moved the symbols.  i removed this from the init